centreon/www/front_src/src/Resources/Actions/index.tsx

```diff
@@ -1,28 +1,74 @@
 import { lazy, Suspense } from 'react';
 
-import { useTheme, Grid } from '@mui/material';
+import { useAtomValue } from 'jotai';
+import { equals } from 'ramda';
+
+import { Grid, useMediaQuery, useTheme } from '@mui/material';
+
+import { featureFlagsDerivedAtom } from '@centreon/ui-context';
 
-import { Props } from './Refresh';
 import GlobalActionsSkeleton from './GlobalActionsSkeleton';
+import { Type } from './model';
+import { Props } from './Refresh';
+import useMediaQueryListing from './Resource/useMediaQueryListing';
 import ResourceActionsSkeleton from './ResourceActionsSkeleton';
+import VisualizationActions from './Visualization';
+import { useStyles } from './Visualization/Visualization.styles';
 
 const WrapperResourceActions = lazy(() => import('./WrapperResourceActions'));
 const GlobalActions = lazy(() => import('./Refresh'));
 
 const Actions = ({ onRefresh }: Props): JSX.Element => {
+  const { classes, cx } = useStyles();
   const theme = useTheme();
 
+  const { applyBreakPoint, breakPointType } = useMediaQueryListing();
+  const { resourceStatusTreeView } =
+    useAtomValue(featureFlagsDerivedAtom) || {};
+  const displayCondensed =
+    Boolean(useMediaQuery(theme.breakpoints.down(1150))) || applyBreakPoint;
+
+  const smallSize =
+    useMediaQuery(theme.breakpoints.down(775)) ||
```

Comment:
As MUI defines [breakpoints](https://mui.com/material-ui/customization/breakpoints/#default-breakpoints), does this work when you replace 775 by 'sm' or 'md'?

Reply:
no :s